qemu-check.exp

```diff
@@ -182,3 +182,8 @@ if {$tests == "all" || $tests == "trusted-keys"} {
 	set basedir [file dirname $argv0]
 	source $basedir/trusted-keys.exp
 }
+if {$tests == "all" || $tests == "rust"} {
+	# Invoke Trusted Keys tests
+	set basedir [file dirname $argv0]
```

Comment:
Please move this up (around line 30?) and remove the similar line in the Trusted Key case above, to avoid duplication